python/pyarrow/tests/test_scalars.py

```diff
@@ -364,6 +367,14 @@ def test_duration_nanos_nopandas():
         arr[0].as_py()
 
 
+def test_month_day_nano_interval():
+    triple = pa.MonthDayNano([3600, 3600, 3600])
```

Comment:
Please use different values for components here.

Reply:
done